backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserResolver.java

```diff
@@ -38,6 +41,19 @@ public List<ApiUserWithStatus> usersWithStatus() {
     return _userService.getUsersAndStatusInCurrentOrg();
   }
 
+  @QueryMapping
+  public Page<ApiUserWithStatus> usersWithStatusPage(
+      @Argument int pageNumber, @Argument String searchQuery) {
+    if (pageNumber < 0) {
+      pageNumber = ApiUserService.DEFAULT_OKTA_USER_PAGE_OFFSET;
+    }
+    if (!searchQuery.isBlank()) {
+      return _userService.searchUsersAndStatusInCurrentOrgPaged(
```

Comment:
Search is being handled by a different service method here only as an intermediate step between now and the rest of the Okta migration. Currently, we have to fetch the entire organization's user list from Okta in order to do this search. Once we switch to getting users from our database, we can have the other method below use the search filter easily.